run-checks

```diff
@@ -246,7 +246,8 @@ if [ "$DEPRECATED" = 1 ]; then
 fi
 
 UNCLEAN="$(git status -s|grep '^??')" || true
-if [ -n "$UNCLEAN" ]; then
+SKIP_UNCLEAN=${SKIP_UNCLEAN=}
```

Comment:
hmm... unless I'm misunderstanding what you want to happen, that should be
```sh
SKIP_UNCLEAN="${SKIP_UNCLEAN:-}"
```
although you could just use it like that in the check,
```sh
if [[ -n "$UNCLEAN" && -z "${SKIP_UNCLEAN:-}" ]]; then
```